config/crd/bases/autoscaling.k8s.io_hvpas.yaml

```diff
@@ -625,6 +625,15 @@ spec:
                                     containerName:
                                       description: Name of the container or DefaultContainerResourcePolicy, in which case the policy is used by the containers that don't have their own policy specified.
                                       type: string
+                                    controlledResources:
+                                      description: Specifies the type of recommendations that will be computed (and possibly applied) by VPA. If not specified, the default of [ResourceCPU, ResourceMemory] will be used.
+                                      items:
+                                        description: ResourceName is the name identifying various resources in a ResourceList.
+                                        type: string
+                                      type: array
+                                    controlledValues:
+                                      description: Specifies which resource values should be controlled. The default is "RequestsAndLimits".
+                                      type: string
```

Comment:
Does it make sense to also specify the `enum` values for these fields, such as `["cpu", "memory"]` and `["RequestsAndLimits", "RequestsOnly"]`? So that HVPA-side validation will fail fast rather than passing it on to VPA and then failing validation?